packages/flutter/lib/src/cupertino/theme.dart

```diff
@@ -232,8 +230,8 @@ class CupertinoThemeData with Diagnosticable {
   ///
   ///  * [MaterialBasedCupertinoThemeData], a [CupertinoThemeData] that defers
   ///    [primaryColor] to its Material [Theme] parent if it's unspecified.
-  Color get primaryColor => _primaryColor ?? _defaults.primaryColor;
-  final Color _primaryColor;
+  Color? get primaryColor => _primaryColor ?? _defaults!.primaryColor;
```

Comment:
Looking at this again, isn't it guaranteed that this will always return a non-null value? primaryColor should always be non-null in the default theme?

Same for many of the getters below that reference the default.

Reply:
Same explanation as https://github.com/flutter/flutter/pull/66024#discussion_r490767475